projects/kit/components/input-date/input-date.component.ts

```diff
@@ -225,10 +225,8 @@ export class TuiInputDateComponent
         return this.activeItem ? `` : filler;
     }
 
-    onMobileClick(): void {
```

Comment:
breaking changes?

Reply:
a method name changed, yes, but other than that it should work as before, I didn't notice any difference